builder/manifest.json

```diff
@@ -0,0 +1,712 @@
+      "layers": [{
+        "script": "builder/img/go.sh",
+        "inputs": ["Godeps/Godeps.json", "builder/go-wrapper.sh"],
+        "limits": { "temp_disk": "1G" }
+      }]
+    },
+    {
+      "id": "builder",
+      "base": "go",
+      "layers": [{
+        "gobuild": {
+          "builder": "/bin/flynn-builder"
+        }
+      }]
+    },
+    {
+      "id": "discoverd",
+      "base": "busybox",
+      "layers": [{
+        "gobuild": {
+          "discoverd": "/bin/discoverd"
+        },
+        "copy": {
+          "discoverd/start.sh": "/bin/start-discoverd"
+        }
+      }],
+      "entrypoint": {
+        "args": [
+          "/bin/start-discoverd"
+        ]
+      }
+    },
+    {
+      "id": "flannel",
+      "base": "busybox",
+      "layers": [{
+        "gobuild": {
+          "flannel":         "/bin/flanneld",
+          "flannel/wrapper": "/bin/flannel-wrapper"
+        }
+      }],
+      "entrypoint": {
+        "args": [
+          "/bin/flannel-wrapper"
+        ]
+      }
+    },
+    {
+      "id": "host",
+      "base": "ubuntu-xenial",
+      "layers": [
+        {
+          "name": "host-packages",
+          "script": "host/img/packages.sh",
+          "limits": { "temp_disk": "1G" }
+        },
+        {
+          "name": "host-binaries",
+          "env": {
+            "TUF_ROOT_KEYS": "{{ .TUFRootKeys }}"
+          },
+          "cgobuild": {
+            "host": "/usr/local/bin/flynn-host"
+          },
+          "gobuild": {
+            "host/flynn-init": "/usr/local/bin/flynn-init"
+          },
+          "copy": {
+            "util/ca-certs/ca-certs.pem": "/etc/ssl/certs/ca-certs.pem",
+            "host/zfs-mknod.sh":          "/usr/local/bin/zfs-mknod",
+            "host/udev.rules":            "/lib/udev/rules.d/10-local.rules",
+            "host/start.sh":              "/usr/local/bin/start-flynn-host.sh",
+            "host/cleanup.sh":            "/usr/local/bin/cleanup-flynn-host.sh"
+          }
+        }
+      ],
+      "entrypoint": {
+        "args": [
+          "/usr/local/bin/start-flynn-host.sh"
+        ]
+      }
+    },
+    {
+      "id": "controller",
+      "base": "busybox",
+      "layers": [{
+        "gobuild": {
+          "controller":           "/bin/flynn-controller",
+          "controller/scheduler": "/bin/flynn-scheduler",
+          "controller/worker":    "/bin/flynn-worker"
+        },
+        "copy": {
+          "controller/start.sh":        "/bin/start-flynn-controller",
+          "util/ca-certs/ca-certs.pem": "/etc/ssl/certs/ca-certs.pem"
+        },
+        "inputs": [
+          "schema/*.json",
+          "schema/controller/*.json",
+          "schema/router/*.json"
+        ],
+        "run": [
+          "mkdir -p /etc/flynn-controller",
+          "cp -r schema /etc/flynn-controller/jsonschema"
+        ]
+      }],
+      "entrypoint": {
+        "args": [
+          "/bin/start-flynn-controller"
+        ]
+      }
+    },
```

Comment:
Does it make more sense to put these manifests in their corresponding component directories similar to how the `Tupfile` used to work?